lib/views/welcome_view.dart

```diff
@@ -150,6 +162,13 @@ class _WelcomeViewState extends State<WelcomeView> with WidgetsBindingObserver {
   }
 
   Widget _buildActionButtons(String? action) {
+    final Map<int, String> useDefaultServerOptions = {
```

Comment:
I'd use an enum instead.

```dart
enum ServerOptions {
    default,
    custom;

    String getLocalized(BuildContext context) {
        var localizations = AppLocalizations.of(context);
        return switch (this) {
            default => localizations.serverOptionDefault,
            custom => localizations.serverOptionCustom,
        };
    }
}
```

Reply:
Although with just two values you might just want to change this into a boolean instead. `bool isCustomServerSelected = false;`. I don't think we'll have more options in the future and even if we do we can still use an enum then.